File: Group-Video/NERtcSample-GroupVideoCall-iOS-Objective-C/NERtcSample-GroupVideoCall-iOS-Objective-C/NTESAppDelegate.m
//
//  NTESAppDelegate.m
//  NERtcGroupMeetingSample
//
//  Created by 丁文超 on 2020/3/23.
//  Copyright © 2020 丁文超. All rights reserved.
//

#import "NTESAppDelegate.h"

@implementation NTESAppDelegate

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
    // 强制触发网络权限请求
    [[NSURLSession.sharedSession dataTaskWithRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://www.baidu.com"]]] resume];
    return YES;
}

@end

File: One-to-One-Video/NERtcSample-1to1-iOS-Swift/NERtcSample-1to1-iOS-Swift/NTESAppDelegate.swift
//
//  NTESAppDelegate.swift
//  NERtcSample-1to1-iOS-Swift
//
//  Created by Wenchao Ding on 2020/7/31.
//  Copyright © 2020 Wenchao Ding. All rights reserved.
//

import UIKit

@UIApplicationMain
class NTESAppDelegate: UIResponder, UIApplicationDelegate {

    var window: UIWindow?
    
    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
        URLSession.shared.dataTask(with: URL(string: "https://www.baidu.com")!).resume()
        return true
    }


}

